src/mame/nintendo/vt1682.cpp

```diff
@@ -240,6 +246,12 @@ class vt_vt1682_state : public driver_device
 	uint8_t m_202e;
 	uint8_t m_2030;
 
+	int m_scu_to_main_irq_active;
+	int m_timera_to_main_irq_active;
+	int m_current_main_vector;
+	int m_new_main_vector;
```

Comment:
Please use explicitly-sized integers for things that need to be registered for saved states to help keep saved states portable.